frontend/public/components/overview/networking-overview.tsx

```diff
@@ -51,7 +51,8 @@ const RoutesOverviewListItem: React.SFC<RoutesOverviewListItemProps> = ({ route
   return (
     <li className="list-group-item">
       <ResourceLink kind="Route" name={name} namespace={namespace} />
-      <span className="text-muted">{t('public~Location:')}</span> <RouteLocation obj={route} />
+      <span className="text-muted">{t('public~Location: ')}</span>
```

Comment:
The space character should be outside then span (not passed to `t` as part of the string to be translated)

Reply:
Fixed by removing the space. Thanks.